invenio_rdm_records/resources/config.py

```diff
@@ -120,6 +119,7 @@ class RDMRecordResourceConfig(RecordResourceConfig, ConfiguratorMixin):
     request_read_args = {
         "style": ma.fields.Str(),
         "locale": ma.fields.Str(),
+        "with_deleted": ma.fields.Bool()
```

Comment:
Very minor:
```suggestion
        "deleted": ma.fields.Bool()
```
of `include-deleted`. The `with`, IMO, is more technical/for code.